doc/Getting_Started.md

```diff
@@ -61,6 +63,9 @@ $ pip3 install --upgrade tockloader
 
 Ubuntu:
 ```
+$ apt install -y build-essential python3-pip curl
```

Comment:
I'm confused. Sure, some of the scripts in `/tools` are written in Python and may require some wheels, but is pip3 really required to build the Tock kernel?

Reply:
`tockloader` requires it, and the instructions immediately below this instruct the user to install `tockloader` using pip3

Reply:
How did I miss that? Thanks :)